packages/core/src/payment/headless-payment-method-response.ts

```diff
@@ -0,0 +1,7 @@
+import HeadlessPaymentMethod from './headless-payment-method';
+
+export interface HeadlessPaymentMethodResponse<T = any> {
```

Comment:
Will it work for you, if `any` be changed with `unknown`?

```suggestion
export interface HeadlessPaymentMethodResponse<T = unknown> {
```

Reply:
Are we not able to define the actul type here?

Reply:
@bc-peng we are using any because of [this comment](https://github.com/bigcommerce/checkout-sdk-js/pull/2754#discussion_r1888442288)